File: python/svix/models/sqs_config.py
# this file is @generated
import typing as t

from .common import BaseModel


class SqsConfig(BaseModel):
    """Configuration for an SQS sink."""

    queue_url: str

    region: str

    access_key_id: str

    secret_access_key: str

    endpoint_url: t.Optional[str] = None

File: kotlin/lib/src/main/kotlin/models/SqsConfig.kt
// This file is @generated
package com.svix.kotlin.models

import kotlinx.serialization.Serializable

@Serializable
data class SqsConfig(
    val queueUrl: String,
    val region: String,
    val accessKeyId: String,
    val secretAccessKey: String,
    val endpointUrl: String? = null,
)
